base/ubi9/Dockerfile

```diff
@@ -77,7 +78,7 @@ RUN \
     echo "export PS1='\W \`git branch --show-current 2>/dev/null | sed -r -e \"s@^(.+)@\(\1\) @\"\`$ '" >> /home/user/.bashrc && \
     # Copy the global git configuration to user config as global /etc/gitconfig
     #  file may be overwritten by a mounted file at runtime
-    cp /etc/gitconfig /home/user/.gitconfig && \
+    # cp /etc/gitconfig /home/user/.gitconfig && \
```

Comment:
If not needed we should remove this line rather than commenting this out.

Reply:
Removed.